### DISCUSSION TITLE
I need help, I am lost.

### DISCUSSION THREAD

**Discussion by kris3713** [supabase-community/supabase-csharp#177] [Q&A] [answered]
I am trying to create a UI application for a final project (in university) that can fetch/set data from Supabase. I have questions that need answering.

Where do I put the initialization client? The documentation does not specify that clearly.

I want to use this package alongside **dotenv.NET**, can you please show me a basic configuration with both of them in use?

If anyone could take the time to help me, I would greatly appreciate it.

**Comment by Jonydeere1** [comment, marked as answer]
its depends on the framework that you are using (MAUI, Blazor, Core, Etc), also the .NET and C# version.
So if your program/app -it's not critical or handle sensible data, etc.- and it's just for basics CRUD operations... i recommend use http connection instead CLI.
